src/components/templates/Jazzbar/JazzTab/JazzTab.tsx

```diff
@@ -51,9 +51,10 @@ interface JazzProps {
 // }
 
 const Jazz: React.FC<JazzProps> = ({ setUserList, venue }) => {
+  // @debt refactor to remove fireStore venue and just use venue from props
```

Comment:
`venue` is already passed as a prop, can you just use that instead and remove the `firestoreVenue`?